test/uk/gov/hmrc/merchandiseinbaggage/service/AuditorSpec.scala

```diff
@@ -34,66 +31,26 @@ import scala.concurrent.Future
 
   private implicit val aHeaderCarrier: HeaderCarrier = HeaderCarrier()
 
   "auditDeclaration" should {
     Seq(Success, Disabled, failed).foreach { auditStatus =>
       s"delegate to the auditConnector and return $auditStatus" in {
 
-        val testAuditConnector = new TestAuditConnector("TestApp") {
-          override def sendResult(path: String, event: JsValue): Future[HandlerResult] = {
-            (event \ "auditSource").get.toString mustBe "\"merchandise-in-baggage-frontend\""
-            (event \ "auditType").get.toString mustBe "\"DeclarationPaymentAttempted\""
-            (event \ "detail").get mustBe toJson(declaration)
-            Future.successful(HandlerResult.Success)
-          }
-        }
+        val mockAuditConnector: AuditConnector = mock(classOf[AuditConnector])
+
+        when(mockAuditConnector.sendExtendedEvent(any())(any(), any()))
+          .thenReturn(Future.successful(auditStatus))
 
         val auditService = new Auditor {
-          override val auditConnector: AuditConnector = testAuditConnector
+          override val auditConnector: AuditConnector = mockAuditConnector
           override val messagesApi: MessagesApi       = aMessagesApi
         }
 
         auditService.auditDeclaration(declaration).futureValue mustBe a[Unit]
       }
     }
-
-    "use DeclarationPaymentAttempted event for amendments" in {
-      val aDeclarationWithAmendment: Declaration = declaration.copy(amendments = List(aAmendment))
-
-      val testAuditConnector: TestAuditConnector = new TestAuditConnector("TestApp") {
-        override def sendResult(path: String, event: JsValue): Future[HandlerResult] = {
-          (event \ "auditSource").get.toString mustBe "\"merchandise-in-baggage-frontend\""
-          (event \ "auditType").get.toString mustBe "\"DeclarationPaymentAttempted\""
-          (event \ "detail").get mustBe toJson(aDeclarationWithAmendment)
-          Future.successful(HandlerResult.Success)
-        }
-      }
-
-      val auditService: Auditor = new Auditor {
-        override val auditConnector: AuditConnector = testAuditConnector
-        override val messagesApi: MessagesApi       = aMessagesApi
-      }
-
-      auditService.auditDeclaration(aDeclarationWithAmendment).futureValue mustBe a[Unit]
-    }
-
-    "handle auditConnector failure" in {
-      val aDeclarationWithAmendment: Declaration = declaration.copy(amendments = List(aAmendment))
-
-      val testAuditConnector: TestAuditConnector = new TestAuditConnector("TestApp") {
-        override def sendResult(path: String, event: JsValue): Future[HandlerResult] =
-          Future.successful(HandlerResult.Failure)
-      }
-
-      val auditService: Auditor = new Auditor {
-        override val auditConnector: AuditConnector = testAuditConnector
-        override val messagesApi: MessagesApi       = aMessagesApi
-      }
-
-      auditService.auditDeclaration(aDeclarationWithAmendment).futureValue mustBe a[Unit]
```

Comment:
Removed both tests as firstly the updated on the branch [here](https://github.com/hmrc/merchandise-in-baggage-frontend/blob/DDCE-5464/test/uk/gov/hmrc/merchandiseinbaggage/service/AuditorSpec.scala#L78-L92) is returning a [success](https://github.com/hmrc/merchandise-in-baggage-frontend/blob/DDCE-5464/test/uk/gov/hmrc/merchandiseinbaggage/service/AuditorSpec.scala#L84) for a failed scenario test but these [two tests](https://github.com/hmrc/merchandise-in-baggage-frontend/blob/DDCE-5464/test/uk/gov/hmrc/merchandiseinbaggage/service/AuditorSpec.scala#L62-L92) have already been covered in the first test. There are no extra checks on the declaration passed so the first one should suffice for both tests.